From c6db8bce4545afd0399e43c0461c9eb99ba7b37d Mon Sep 17 00:00:00 2001
From: Zhu Yuqing <b48954@freescale.com>
Date: Tue, 13 May 2014 13:19:03 +0800
Subject: [PATCH 063/115] ENGR00303286 fix OTA: The pop up menu size is
 changed when check new version and display the
 version name

change the android:layout_height="wrap_content" into layout_height="178dp"

Signed-off-by: Zhu Yuqing <b48954@freescale.com>
---
 FSLOta/res/layout/main.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FSLOta/res/layout/main.xml b/FSLOta/res/layout/main.xml
index b102697..592a258 100644
--- a/FSLOta/res/layout/main.xml
+++ b/FSLOta/res/layout/main.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
+    android:layout_height="178dp"
     android:baselineAligned="false"
     android:gravity="center|center_horizontal|center_vertical"
     android:orientation="horizontal" >
-- 
1.8.0

